docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -39,16 +39,18 @@ dynamicMetadata: {
   }
 }
 ```
+or:
+```go
+dynamicMetadata: {
+  "envoy.lb": {
+    "x-gateway-destination-endpoint": <ip:port>,<ip:port>,...
+  }
+}
+```
 
-Constraints:
-- If the EPP did not communicate the server endpoint via these two methods, it MUST return an error as follows:
-  -  [ImmediateResponse](https://github.com/envoyproxy/envoy/blob/f2023ef77bdb4abaf9feef963c9a0c291f55568f/api/envoy/service/ext_proc/v3/external_processor.proto#L195) with 503 (Serivce Unavailable) HTTP status code if there are no ready endpoints.
-  -  [ImmediateResponse](https://github.com/envoyproxy/envoy/blob/f2023ef77bdb4abaf9feef963c9a0c291f55568f/api/envoy/service/ext_proc/v3/external_processor.proto#L195) with 429 (Too Many Requests) HTTP status code if the request should be dropped (e.g., a Sheddable request, and the servers under heavy load).
-- The EPP MUST not set two different values in the header and the inner response metadata value. 
-- Setting different value leads to unpredictable behavior because proxies aren't guaranteed to support both paths, and so this protocol does not define what takes precedence.
+The value of the header or metadata entry MUST contains at least one endpoint in `<ip:port>` format or multiple endpoints in `<ip:port>,<ip:port>,...` format. Multiple endpoints are separated by commas. The first valid endpoint in the value will be used. And if retrying is happening, the proxy will try the endpoints after the previously selected endpoint in order.
```

Comment:
nit: s/contains/contain/

Reply:
> The value of the header or metadata entry  is the expected behavior if both are set?

In the previous EPP proposal, the behavior is not defined. In the override host lb, if both are set, the list will be concated in the specified order.